### ISSUE TITLE
Other Language doesn't generate (zh-cn, ja) 

### ISSUE THREAD

**Issue by MaiChaMH** [lukaszliniewicz/Pandrator#15]
I'm using the 0.0.7 version with xtts (installed through one click exe). English and some other languages seem to be doing fine. However, when I select another language (zh-cn & ja specifically) it doesn't generate. 

There's nothing happening after pressing the Start generation button, and to confirm I clicked the cancel generation button after the start button it says that no process is happening (also went to task manager to double check, no CPU process).

**Comment by lukaszliniewicz** [comment]
Sorry about that, I think I know what's wrong - the sentence splitting library I'm using does not support either Chinese or Japanese. I have to use a different one for these languages. I will post an update soon.

**Comment by lukaszliniewicz** [comment]
Please try installing it again (remove the Pandrator folder first). Japanese should work now, but I'm not sure if splitting sentences into shorter chunks based on the character limit set under "Text processing" works well or how many Japanese characters XTTS can process well in one go. Please let me know what you find.

**Comment by MaiChaMH** [comment]
> Please try installing it again (remove the Pandrator folder first). Japanese should work now, but I'm not sure if splitting sentences into shorter chunks based on the character limit set under "Text processing" works well or how many Japanese characters XTTS can process well in one go. Please let me know what you find.

I've re-installed it (by removing the folder). Japanese tts generation is working fine now. 

In terms of Text Processing, I think it's working fine. I've fed it a txt with around 4000 Japanese characters, with paragraph spacing and without (paragraph spacing works fine). I've tested length 30 and length 3000, with 30 it'd successfully split the end of a sentence, with 3000 XTTS handled it in one go with no problems and no artifacts in the audio (very powerful).

![image](https://github.com/lukaszliniewicz/Pandrator/assets/123769025/04342b9d-5ac5-4336-afb1-a9871a5f035f)

Also for Chinese libraries, I've found (Asking Bing AI) these 2 that might be helpful. pywordseg and jieba library, according to the search results from the AI, it seemed that pywordseg for word segmentation, and jieba can do both word segmentation and sentence splitting. (Both libraries can do simplified and traditional Chinese characters.)

![image](https://github.com/lukaszliniewicz/Pandrator/assets/123769025/8d8c666c-d5b1-4bb2-8351-76e4e66a4337)
